src/containers/Sidebar/SidebarNodeView.tsx

```diff
@@ -31,13 +31,14 @@ class NodeDrawer extends React.Component<Props, State> {
   readonly state: Readonly<State> = {};
 
   selectParentClicked = async () => {
-    const { modelId, revisionId, nodeId, rootAssetId } = this.props.app;
+    const { modelId, revisionId, nodeId } = this.props.app;
     const parent = await sdk.revisions3D.list3DNodes(modelId!, revisionId!, {
       nodeId,
     });
     if (parent.items.length > 0) {
       this.props.setModelAndRevisionAndNode(
-        rootAssetId!,
+        modelId!,
+        revisionId!,
```

Comment:
Do we need the `!`?

Reply:
yes because by default the app state's modelId may be null

Reply:
it would never be null here though :P there's checks done before component is loaded.